handlers/spaces.py

```
@@ -672,19 +682,16 @@ async def shorten_summary(update: Update, context: ContextTypes.DEFAULT_TYPE):
     # Use the new helper function to process the response
     await process_summary_api_response(context, update, edit_response, processing_msg)
 
+def validate_request_type(request_type: str) -> bool:
+    """Validate the request type is either 'text' or 'audio'."""
+    return request_type.lower() in ('text', 'audio')
+
 async def summarize_space(update: Update, context: ContextTypes.DEFAULT_TYPE):
     """Handle /summarize_space command with improved error handling."""
     try:
         # Check if the user is a member using find_member_by_username
         username = update.effective_user.username
```

Ruff (0.8.2)

Local variable `username` is assigned to but never used

Remove assignment to unused variable `username`

(F841)